pvanalytics/quality/gaps.py

```diff
@@ -119,7 +114,6 @@ def stale_values_diff(x, window=6, rtol=1e-5, atol=1e-8, mark='tail'):
     """
     if window < 2:
         raise ValueError('window set to {}, must be at least 2'.format(window))
-
```

Comment:
Were all these blank lines removed intentionally?  I think I'd keep all of them in, personally

Reply:
That was a mistake on my part! Added those guys back in